assets/js/instructions-persist-confirm-allocation.js

```diff
@@ -0,0 +1,13 @@
+window.addEventListener('load', function () {
```

Comment:
I have separated this from the pop page scripts so we can just purge them here.

But IMO this is much cleaner